### PR TITLE
New development creation experience

### LINKED ISSUE #105: Refine UX for inputting location for developments
Per my conversation with Tim:

- Add location by clicking on the map
- Refine location originally entered as address
- Lat, lng is the source of truth, not the address
- Users should have to confirm points not entered in parcel

### PR DESCRIPTION
**Pull request by iangmhill**
In this new development creation experience:

1. Users select the location of their development by dragging the map
2. The selected location on the map shows the parcel the location is on top of
3. Latitude and longitude become the primary fields for location and address and zip_code become derivative.

This partially resolves #105. A "goto" search box to help users zoom around the map would further improve this experience but does not affect the ultimate development creation functionality.

Testing this branch will require a migration to load the parcel data into your database.

### CODE REVIEW

File: rails/app/controllers/developments_controller.rb [hunk 1]
```diff
@@ -55,7 +55,10 @@ def create
         format.jsonapi { render json: FullDevelopmentSerializer.new(@development).serialized_json }
       end
     else
-      render json: FullDevelopmentSerializer.new(@development.errors.full_messages).serialized_json, status: :bad_request
+      puts @development.errors.full_messages
```
Comment (mzagaja) [inline rails/app/controllers/developments_controller.rb]: Was this for temporary debugging? Could we still find a way to respond with the error to the client and not simply a `head :bad_request`?

File: ember/app/styles/partials/_vars.scss [hunk 2]
```diff
@@ -19,6 +19,11 @@ $color_font-dark: #091F2F;
...
 $color_state-negative--dark: darken($color_state-negative, 15);
 
+$color_button-disabled: #b3b3b3;
+
 $font_family-primary: 'Nunito', Helvetica, sans-serif;
 
 $transition_border: border-color .25s;
+
+$development_font_size-h1: 26px;
+$development_font_size-h2: 18px;
```
Comment (ericyoungberg) [inline ember/app/styles/partials/_vars.scss]: To reduce indirection, my typical convention for this file is to globally scope generic constants that don't have any affiliation to a single piece of the app. I'd place these two variables at the top of the file which they are being used, scoped inside of that file's wrapping class. 

```scss
.component.development-form {
    $font_size-h1: 26px;
    $font_size-h2: 18px;

    ...
```

File: ember/app/templates/components/development-form.hbs [hunk 3]
```diff
@@ -11,32 +11,56 @@
...
+        <label for="development_latitude" class="hidden">Latitude</label>
+        {{input
+          id="development_latitude"
+          value=(to-fixed editing.latitude 6)
+          placeholder="Latitude"
+          readonly="readonly"
+        }}
+        <label for="development_longitude" class="hidden">Longitude</label>
+        {{input
+          id="development_longitude"
```
Comment (ericyoungberg) [inline ember/app/templates/components/development-form.hbs]: Can we remove the id?

File: ember/app/templates/components/development-form.hbs [hunk 4]
```diff
@@ -11,32 +11,56 @@
...
+          placeholder="Development Name"
+          key-up=(action checkForUpdated 'name')
+        }}
+      </div>
+
+      <div class="form-row required secondary location-fields">
+        <img src="/assets/images/map-pin.svg">
+        <label for="development_latitude" class="hidden">Latitude</label>
+        {{input
+          id="development_latitude"
```
Comment (ericyoungberg) [inline ember/app/templates/components/development-form.hbs]: Can we remove the id?

File: ember/app/templates/components/development-form.hbs [hunk 5]
```diff
@@ -11,32 +11,56 @@
...
-        {{input value=editing.address placeholder="Development Street Address" key-up=(action checkForUpdated 'address')}}
-        {{input value=editing.zipCode placeholder="Zip Code" key-up=(action checkForUpdated 'zipCode')}}
-      </h2>
-
-      <h2 class="project-url">
+
+      <div class="form-row required primary">
+        <label for="development_name" class="hidden">Development Name</label>
+        {{input
+          id="development_name"
```
Comment (ericyoungberg) [inline ember/app/templates/components/development-form.hbs]: Can we remove the id?

**Comment by mzagaja** [comment]
In this branch when attempting to log in with a username and password that does not exist rails returns an HTTP 500 error but ember fails to parse it and hangs at "logging in".  My console errors are enumerated below.
```
[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (sign_in, line 0)
[Error] JSON Parse error: Unrecognized token '<'
parse@[native code]
tryCatcher@http://127.0.0.1:4200/assets/vendor.js:75670:26
invokeCallback@http://127.0.0.1:4200/assets/vendor.js:75848:33
publish@http://127.0.0.1:4200/assets/vendor.js:75834:23
invoke@http://127.0.0.1:4200/assets/vendor.js:32061:28
flush@http://127.0.0.1:4200/assets/vendor.js:31940:31
flush@http://127.0.0.1:4200/assets/vendor.js:32132:36
end@http://127.0.0.1:4200/assets/vendor.js:32242:47
_boundAutorunEnd@http://127.0.0.1:4200/assets/vendor.js:32204:26
	error
	defaultDispatch (vendor.js:49378)
	dispatchError (vendor.js:49359)
	onerrorDefault (vendor.js:59471)
	trigger (vendor.js:75530)
	(anonymous function) (vendor.js:76361)
	invoke (vendor.js:32061)
	flush (vendor.js:31940)
	flush (vendor.js:32132)
	end (vendor.js:32242)
	_boundAutorunEnd (vendor.js:32204)
```

Steps to reproduce:
1. Pick an arbitrary username/password that both do not exist in the database.
2. Attempt to login .
3. Receive HTTP 500 error and notice that the spinner continues spinning without user feedback.

File: ember/app/templates/components/development-form.hbs (hunk 3 above)
Comment (iangmhill) [reply ember/app/templates/components/development-form.hbs]: The `id` and `for` attributes are required to link the label and input for accessibility. Ideally this pattern would extend across the whole form, but that would have required more refactoring.

**Comment by iangmhill** [comment]
Fixed the login bug for nonexistent users.

**Pull request merged**